### DISCUSSION TITLE
Final Project Repository

### DISCUSSION THREAD

**Discussion by jrtib** [comp426-2023-spring/syllabus#33] [General]
For the final project, will there be a repository released through GitHub Classroom for each team or is it ok if we make a public repository for the project for our specific team and then invite teammates as collaborators to the repository?

**Comment by sadieamato** [comment]
You can use the a99 repo as a template and create it yourself! See my [issue](https://github.com/comp426-2023-spring/a99/issues/4) for more detailed instructions on how we're supposed to do this.
